presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -64,7 +63,7 @@ public TableScanStatsRule(Metadata metadata, StatsNormalizer normalizer)
         for (Map.Entry<Symbol, ColumnHandle> entry : node.getAssignments().entrySet()) {
             Symbol symbol = entry.getKey();
             Optional<ColumnStatistics> columnStatistics = Optional.ofNullable(tableStatistics.getColumnStatistics().get(entry.getValue()));
-            outputSymbolStats.put(symbol, columnStatistics.map(statistics -> toSymbolStatistics(tableStatistics, statistics)).orElse(UNKNOWN_STATS));
+            outputSymbolStats.put(symbol, columnStatistics.map(statistics -> toSymbolStatistics(tableStatistics, statistics)).orElse(SymbolStatsEstimate.unknown()));
```

Comment:
`orElseGet(SymbolStatsEstimate::unknown)`

Reply:
What's the difference?

Reply:
Should `.unknown()` return new instance each time its called (should we assume here it does not do this?), `orElse(SymbolStatsEstimate.unknown())` allocates this new object while `orElseGet(SymbolStatsEstimate::unknown())` uses a constant (lambda), delaying instantiation until needed.

(of course, this isn't much of a difference, more a matter of style)